src/nedap/utils/test/impl.cljc

```diff
@@ -40,3 +41,16 @@
        (if (cljs.test/successful? summary)
          (set! (.-exitCode js/process) 0)
          (set! (.-exitCode js/process) 1)))))
+
+(defn expect
+  [bodies {:keys [to-change from to]} clj?]
+  {:pre [(spec/valid? seq bodies)
+         (spec/valid? some? to)
+         (spec/valid? some? from)
+         (spec/valid? some? to-change)
```

Comment:
Additional precondition: `(not= from to)`

Pay attention to metadata: `:from ^:a [] :to ^b []` is acceptable, `:from [] :to []` is not

Reply:
This'd break functionality. it might be a valid usecase to expect a certain value does not change after evaluating bodies

Reply:
`:to-change` literally says "to change", not "not to change".

The described use case would be a separate feature - I wouldn't weaken the original feature for supporting that one (especially when `(expect (x) :to-change @a :from 0 :to 0)` doesn't make sense from an English POV: things change to _something else_)

It is predictable that in a future, `expect` can become polymorphic, supporting `:to-not-change` and other "predicates".

Step by step :)

Reply:
That would also allow us to change the syntax to be more specific; `(expect () :not-to-change @a)` (optionally with a starting-value even).

I was also thinking about a 'dynamic' change, e.g. `(expect (swap inc a) :to-change @a :by 1)`. also for a different PR obv.